packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -678,17 +678,15 @@ export class EngineReportingAgent<TContext = any> {
     executableSchemaId: string;
     executableSchema: string;
   }) {
-    this.logger.info('Starting schema reporter...')
-
-    this.logger.info(
-      `Schema reporter options: ${JSON.stringify(
-        {
-          overrideReportedSchema: this.options.experimental_overrideReportedSchema,
-          schemaReportingInitialDelayMaxMs: this.options.experimental_schemaReportingInitialDelayMaxMs,
-          schemaReportingUrl: this.options.schemaReportingUrl,
-        }
-      )}`
-    )
+    this.logger.info('Starting schema reporter...');
+    this.logger.info(`Schema reporting override is ${
+      this.options.experimental_overrideReportedSchema ?
+        'enabled' : 'disabled'
+    }`);
+    this.logger.info(`Schema reporting max initial delay: ${
+        this.options.experimental_schemaReportingInitialDelayMaxMs
+      } ms`);
+    this.logger.info(`Schema reporting URL: ${this.options.schemaReportingUrl}`);
```

Comment:
Ditto to above. I would check if its undefined.

Reply:
Same as above, added the undefined check and change the phrasing to `Schema reporting URL override`